changelog/load-balancer/february2022/updated-failover-mechanism.mdx

```diff
@@ -0,0 +1,12 @@
+---
+title: Load Balancer has improved its Failover mechanism.
+status: updated
+author:
+    fullname: Load Balancer Team
+    url: ''
+date: 2022-02-02
+category: network
+product: network_lb
+---
+
+A pool of instances stands ready to take over in case of failure. More information is available in our [blog post](https://blog.scaleway.com/strengthening-our-load-balancer-failover-mechanism/).
```

Comment:
```suggestion
A pool of Instances stands ready to take over in case of failure. More information is available in our [blog post](https://blog.scaleway.com/strengthening-our-load-balancer-failover-mechanism/).
```